web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -12,7 +12,7 @@ interface State {
   isAuthenticated: boolean
   tokenRefreshed: boolean
   token: string | undefined
-  roles: string[]
+  shouldEnableFireStarts: boolean
```

Comment:
I'm worried this is too specific on this level. Aren't we better off having `roles: string[]` ? When we start adding in other roles, for other products, it could get very confusing. e.g. if you're busy working on MoreCast, what relevance does `shouldEnableFireStarts` have on the global authentication slice?

Reply:
I'm worried that's too general! I say we expose only what's needed. This also removes the concern you had in the last PR around if `isAuthenticated` is true or not.

Reply:
shouldEnableFireStarts relates only to HFI though, and that's not clear looking at this.

Reply:
Good point, I was thinking of throwing these in a `permissions` object within the store. That way it's namespaced.